### PR TITLE
Fixing Servant cookbooks part 1 (-testing + sqlite-simple)

### PR DESCRIPTION
**Pull request by akhesaCaro**
The scope of this PR, as the others that will follow, is to fix servant cookbooks.

This PR make the project compiling and propose a first step through that goal.

Here are the tasks that have been made : 

- Fixing the network bug
- Commenting servant-quickcheck cookbook which is trapped into a dependency loop for now (see [this issue](https://github.com/haskell-servant/servant-quickcheck/issues/74))
- Fixing DocSpec failing test
- Bringing sqlite-simple cookbook back to life

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -55,8 +55,6 @@ constraints: base-compat ^>=0.11
 constraints: semigroups ^>=0.19
 
 -- MonadFail
```
Comment (gdeest) [inline cabal.project:57]:
```suggestion
```

This line should be removed as well.

File: .gitignore [hunk 2]
```diff
@@ -30,6 +30,7 @@ doc/_build
 doc/venv
 doc/tutorial/static/api.js
 doc/tutorial/static/jq.js
+shell.nix
```
Comment (gdeest) [inline .gitignore:33]: Not strictly part of this PR.

File: cabal.project (hunk 1 above)
Comment (akhesaCaro) [reply cabal.project]: :+1:

**Review by gdeest — APPROVED** [review]
Looks good ! :)

**Pull request merged**